ARAnalytics.m

```diff
@@ -420,7 +433,7 @@ + (void)event:(NSString *)event {
 }
 
 + (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
```

Comment:
Not Sure this is a good place to do this (inside the ARAnalytics I mean). Outside of the AR we add super properties using addSuperProperties on WMLAnalyticsService, which feels more right to me. Maybe you can put an RAC hook on the event calling, or on the time change (every hour) to do it instead).  @zats  What do you think?